drill/README.md

```diff
@@ -50,6 +69,10 @@ apache drill 1.9.0
 12 rows selected (3.943 seconds)
 ```
 
+### On single node clusters
+
+In order to use Drill on single node cluster, run `usr/lib/drill/bin/drill-embedded` or run sqlline with zk=local: `/usr/lib/drill/bin/sqlline -u jdbc:drill:zk=local`.
```

Comment:
Neither DRILL_LOG_DIR nor DRILL_YARN_LOG_DIR seem to help to run as my user (not `root`). You may need to look into the source code of those commands.

```
$ /usr/lib/drill/bin/sqlline -u jdbc:drill:zk=local
ERROR: Log directory does not exist or is not writable: /usr/lib/drill/log
```

If that isn't a quick fix, let's just document these commands with `sudo`.